agent/www/shared/address_service.js

```diff
@@ -250,19 +301,13 @@ AddressService.prototype.update_user = function (c) {
 
 AddressService.prototype.on_message = function (context) {
     if (context.message.subject === 'address') {
-        this.update(context.message.body);
+        this.update_address(context.message.body);
         if (this.callback) this.callback('address');
```

Comment:
I don't see the point of the `callback("address")`. By this point we will have already dispatched a `address_added` or `address_updated`, so why the bare `address` too?